elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -98,6 +125,24 @@ $LOCAL-VARIABLES: (
 				margin-top: 0 !important;
 			}
 		}
+		:host([pfe-media]) & {
```

Comment:
This can be updated to remove prefixing.  What would you think of `is-media` or `has-media`?  Active attributes make the markup very readable.